src/Compilers/Core/Portable/SpecialMember.cs

```diff
@@ -144,6 +144,8 @@ internal enum SpecialMember
         System_Nullable_T__op_Implicit_FromT,
         System_Nullable_T__op_Explicit_ToT,
 
+        System_Runtime_CompilerServices_RuntimeFeature__DefaultImplementationsOfInterfaces,
```

Comment:
> Just curious: why use a special member rather than a well-known member?

Members of special types are special members

---
In reply to: [250671932](https://github.com/dotnet/roslyn/pull/32689#discussion_r250671932) [](ancestors = 250671932)